### ISSUE TITLE
_NEED_ to SWITCH From J.R. "Premium" to "Free"—Cannot Find the Way to Do that!!

### ISSUE THREAD

**Issue by Chuck-7** [ZachSaucier/Just-Read#317]
Could you __PLEASE__ Either::
(1) CANCEL my Subscription to JustRead "Premium";; To Switch me TO:: JustRead FREE!!;; __OR__
(2) Let me know HOW to CANCEL JustRead "Premium".

THANK YOU VERY MUCH for your _HELP_!!
Chuck Colsch

**Comment by ZachSaucier** [comment]
Hey Chuck, thanks for reaching out.

You can learn how to unsubscribe in the FAQ section for Just Read
<https://github.com/ZachSaucier/Just-Read#faq> under "How can I unsubscribe
from Just Read Premium?". Alternatively, you can send me the receipt and I
can cancel the subscription.

May I ask, is there anything I can do to help you keep your subscription
active? What is it that is causing you to cancel?

**Comment by Chuck-7** [comment]
THANK YOU, Zach, for your Quick Reply!!
Actually, no, It was working GREAT!
I simply noticed that in actual practice, that I find myself nearly always just clicking on the (Native) Reader Mode {simply because it seems that LATELY I am Constantly Rushing, Rushing).
HOWEVER, “Just Read” is surely the BEST of the nicer Readers, ’cause I designed all of my page elements so beautifully.
Thank you AGAIN for replying so quickly!!
–Chuck
